src/components/Navbar/Navbar.tsx

```diff
@@ -0,0 +1,38 @@
+import { NavLink } from 'react-router-dom';
+import classNames from 'classnames';
+
+export const Navbar = () => {
+  const getLinkClass = ({ isActive }: { isActive: boolean }) => {
+    return classNames('navbar-item', {
+      'has-background-grey-lighter': isActive,
+    });
+  };
+
+  return (
+    <nav
+      data-cy="nav"
+      className="navbar is-fixed-top has-shadow"
+      role="navigation"
+      aria-label="main navigation"
+    >
+      <div className="container">
+        <div className="navbar-brand">
+          <NavLink
+            className={getLinkClass}
+            to="/"
+          >
+            Home
+          </NavLink>
+
+          <NavLink
+            aria-current="page"
+            className={getLinkClass}
+            to="/people"
+          >
+            People
+          </NavLink>
```

Comment:
better create array with object, it's gonna be easier in future to expand list of links